databases/spatialite-tools/Portfile

```diff
@@ -3,35 +3,36 @@
 PortSystem 1.0
 
 name                    spatialite-tools
-version                 5.0.0
+version                 5.1.0
 revision                0
 categories              databases gis
 license                 GPL-3
 platforms               darwin
```

Comment:
this is the default and the line can this be removed

Reply:
removed